server.js

```diff
@@ -260,6 +260,10 @@ function makeCreateOrImportRepo (mode) {
         );
     };
 }
+// GITHUB APIs
+router.get("/api/get-repo/:owner/:shortName", ensureAPIAuth, loadGH, function (req, res) {
```

Comment:
a nit, but I think `/api/repos/:owner/:shortName` would better match the existing `edit` route

Reply:
Route updated in https://github.com/w3c/ash-nazg/pull/163/commits/c5291ccd6ce483b2e738e7c2d15962dae6776e19